jquery.h5validate.js

```diff
@@ -161,6 +165,66 @@
 			}, validity);
 		},
 
+		/**
+		 * Add builtin validators to h5Validate. Currently, this adds the following validators:
+		 *   Required
+		 *   Maxlength
+		 *   Pattern
+		 * @param  {object}  settings   instance settings
+		 */
+		addBuiltinValidators = function (settings) {
```

Comment:
Is there a good reason to distinguish between "built in" validators and add-on / plugin validators?

It might be useful to treat them all the same way...